libc/hdr/types/CMakeLists.txt

```diff
@@ -63,3 +63,43 @@ add_proxy_header_library(
     libc.include.llvm-libc-types.fexcept_t
     libc.include.fenv
 )
+
+add_proxy_header_library(
+  time_t
+  HDRS
+    time_t.h
+  FULL_BUILD_DEPENDS
+    libc.include.llvm-libc-types.time_t
```

Comment:
Does it need `libc.include.time` also?  Similarly for the other type headers here.